### ISSUE TITLE
calculator selection via args

### ISSUE THREAD

**Issue by mpwaller** [qrefine/qrefine#9]
In line with the command line argument:
data_weight=0

we want to have one for qm calculator, e.g. default:
calculator=gaussian

**Comment by zhengmin317** [comment]
Running qm calculation by wrapper is a temporary choice.
now some people are using gaussian, we add gaussian as a calculator.
later people want to use NWCHEM or ADF, then we need to add NWCHEM or ADF as a calculator.
Then our code QR is affected by the QM software people are using.

P.S The tight integration will be our focus as we have discussed and agreed. we may choose a python qm software, like PyQuante or Serenity, after we get nice result from p26 calculation.

**Comment by mpwaller** [comment]
I agree with what you say, but we can support this ASE integration with a calculator chosen as a command line argument in the meantime. I will at this later today.

**Comment by hokru** [comment]
closed with b7c0cbc
